.teamcity/_self/projects/WPComTests.kt

```diff
@@ -137,7 +140,18 @@ fun gutenbergCoreE2eBuildType(): BuildType {
 					npm run test:e2e:playwright
 				""".trimIndent()
 				dockerImage = "%docker_image_ci_e2e_gb_core_on_dotcom%"
+				dockerRunParameters = "-u %env.UID% --log-driver=json-file --log-opt max-size=10m --log-opt max-file=3"
 			}
+
+			step(ScriptBuildStep {
+				name = "Copy Docker Logs"
+				scriptContent = """
```

Comment:
This is an attempt to capture the logs to troubleshoot the issue I'm having with docker in CI, but I'm a bit blind atm here, I'm not sure if this will work; I might need to iterate further.